### DISCUSSION TITLE
typeorm and next.js configuration

### DISCUSSION THREAD

**Discussion by spotykatch** [swc-project/swc#7343] [General]
I'm working on this issue [#9959](https://github.com/typeorm/typeorm/issues/9959) with typeorm. This [pr](https://github.com/typeorm/typeorm/pull/9962) sims to solve the problem with webpack alone, but it does not with next.js. So I thought that SWC is not caring about browser field inside project.json? How can I make it work, or maybe there is another reason?

**Comment by kdy1** [comment]
SWC is not a bundler, so it does nothing to do with the `browser` field

**Reply by spotykatch** [reply]
Can you, please explain, why it works [here](https://github.com/spotykatch/browser-example) and does not with next.js? Is that about next.js? The only change there is browser field.
